scripts/release

```diff
@@ -4,12 +4,13 @@ set -e
 source $(dirname $0)/lib/debug_functions
 
 DOCKER_TAG=${1:-latest}
+REPO=${REPO:-quay.io/submariner}
 
 echo "$QUAY_PASSWORD" | docker login quay.io -u "$QUAY_USERNAME" --password-stdin
-docker tag rancher/submariner:dev quay.io/submariner/submariner:${DOCKER_TAG#"v"}
-docker tag rancher/submariner:dev quay.io/submariner/submariner:"${TRAVIS_COMMIT:0:7}"
-docker tag rancher/submariner-route-agent:dev quay.io/submariner/submariner-route-agent:${DOCKER_TAG#"v"}
-docker tag rancher/submariner-route-agent:dev quay.io/submariner/submariner-route-agent:"${TRAVIS_COMMIT:0:7}"
-docker tag rancher/dapper-base:dev quay.io/submariner/dapper-base:${DOCKER_TAG#"v"}
-docker tag rancher/dapper-base:dev quay.io/submariner/dapper-base:"${TRAVIS_COMMIT:0:7}"
-for i in $(docker images --format "{{.Repository}}:{{.Tag}}" | grep "quay.io/submariner"); do docker push $i; done
+docker tag ${REPO}/submariner:dev ${REPO}/submariner:${DOCKER_TAG#"v"}
+docker tag ${REPO}/submariner:dev ${REPO}/submariner:"${TRAVIS_COMMIT:0:7}"
+docker tag ${REPO}/submariner-route-agent:dev ${REPO}/submariner-route-agent:${DOCKER_TAG#"v"}
+docker tag ${REPO}/submariner-route-agent:dev ${REPO}/submariner-route-agent:"${TRAVIS_COMMIT:0:7}"
+docker tag ${REPO}/dapper-base:dev ${REPO}/dapper-base:${DOCKER_TAG#"v"}
+docker tag ${REPO}/dapper-base:dev ${REPO}/dapper-base:"${TRAVIS_COMMIT:0:7}"
+for i in $(docker images --format "{{.Repository}}:{{.Tag}}" | grep -E "${REPO}/.*:(${DOCKER_TAG#v}|${TRAVIS_COMMIT:0:7})"); do docker push $i; done
```

Comment:
Does the grep -E is really required? Seems like grep "quay.io/submariner" and grep -E "${REPO}/.*:(${DOCKER_TAG#v}|${TRAVIS_COMMIT:0:7})" return the same results that include all tags not just latest and commit hash. Maybe grep -E "${REPO}/.*:(${DOCKER_TAG#v}|\${TRAVIS_COMMIT:0:7})" will return just the required tags.

Reply:
:) github remove the \ character  before $,  i meant grep -E "${REPO}/.:(tag|\tag)"

Reply:
On my system:

```
$ docker images --format "{{.Repository}}:{{.Tag}}" | grep "quay.io/submariner"
quay.io/submariner/submariner-route-agent:latest
quay.io/submariner/submariner:latest
quay.io/submariner/submariner-operator:dev
quay.io/submariner/submariner-operator:0.0.1
```

whereas

```
$ docker images --format "{{.Repository}}:{{.Tag}}" | grep -E "quay.io/submariner/.*:(latest|blah)"
quay.io/submariner/submariner-route-agent:latest
quay.io/submariner/submariner:latest
```

Reply:
Yes you are correct i didn't set TRAVIS_COMMIT and it returned all tags.